api/CredentialsContainer.json

```diff
@@ -493,7 +493,7 @@
                   "chrome_android": "mirror",
                   "edge": "mirror",
                   "firefox": {
-                    "version_added": false
+                    "version_added": "139"
```

Comment:
@Elchi3 According to https://bugzilla.mozilla.org/show_bug.cgi?id=1844449 support was also added for the [credProtect](https://developer.mozilla.org/en-US/docs/Web/API/Web_Authentication_API/WebAuthn_extensions#credprotect) extension. Don't know why tests would not have shown this.

Reply:
Thanks @hamishwillee 👍